src/mainwindow.cpp

```diff
@@ -420,11 +420,11 @@ void MainWindow::setAlbumImage(const lib::spt::entity &albumEntity,
 	const std::string &albumImageUrl)
 {
 	Http::getAlbum(albumImageUrl, *httpClient, cache,
-		[this, albumEntity](const QPixmap &image)
+		[this, albumEntity](const std::string &data)
```

Comment:
Any specific reason for switching this out to a string?

Reply:
Yea, I did this because I added an overload method `setAlbumJpg` farther down the call tree that takes in a string and converts it to a pix map. But I think with your suggestions this change not be needed, if `Context::Content::Content` can have like a `scaleSize` method to update it's album art's size.

Reply:
Yea, I did this because I added a method setAlbumJpg farther down the call tree, and so I added a bunch of overloaded methods on the way up. Agreed, that's probably not the best way to do it - lots of code duplication for no good reason.

But I think with your suggestions this change will not be needed, if Context::Content::Content can have like a scaleSize method to update it's album art's size.

I'm hoping to refactor in such a way so that way I can revert this change.